src/robot_dart/talos.cpp

```diff
@@ -472,6 +468,24 @@ int main(int argc, char* argv[])
                                 << controller->rf_force_filtered().transpose() << std::endl;
                 else if (x.first == "momentum") // the momentum according to pinocchio
                     (*x.second) << controller->momentum().transpose() << std::endl;
+                else if (x.first == "cop") { // the cop according to controller
+                    if (controller->cop())
+                        (*x.second) << controller->cop().value().transpose() << std::endl;
+                    else
+                        (*x.second) << Eigen::Vector2d::Constant(1000).transpose() << std::endl;
+                }
+                else if (x.first == "lcop") { // the cop according to controller
+                    if (controller->lcop())
+                        (*x.second) << controller->lcop().value().transpose() << std::endl;
+                    else
+                        (*x.second) << Eigen::Vector2d::Constant(1000).transpose() << std::endl;
+                }
+                else if (x.first == "rcop") { // the cop according to controller
+                    if (controller->rcop())
+                        (*x.second) << controller->rcop().value().transpose() << std::endl;
+                    else
+                        (*x.second) << Eigen::Vector2d::Constant(1000).transpose() << std::endl;
+                }
                 else
                     (*x.second) << robot->body_pose(x.first).translation().transpose() << std::endl;
```

Comment:
we can pass string that are no bodies using the -l option. this lead to a warning output (at each time step)
should be better to check if robot->body_node(x.first) is not nullptr

Reply:
Proposed FIX in #79